test/spec/components/test/context.test.jsx

```diff
@@ -134,6 +136,45 @@ describe('<TestContext />', () => {
       expect(snippet).to.have.lengthOf(0);
       expect(img).to.have.lengthOf(1);
     });
+
+    it('renders local video', () => {
+      const context = '/testvideo.mp4';
+      const { wrapper, snippet, video } = getInstance({
+        context: JSON.stringify(context),
+        className: 'test'
+      });
+      expect(wrapper).to.have.className('test');
+      expect(snippet).to.have.lengthOf(0);
+      expect(wrapper).to.have.descendants('video');
```

Comment:
I did `expect(wrapper).to.have.descendants('video')` instead of `expect(video).to.have.lengthOf(1)` for the failure case.

```js
expect(video).to.have.lengthOf(1);
// failure: Expected 0 to be 1

expect(wrapper).to.have.descendants('video');
// failure: expected <TestContext /> to have descendants 'video' (and then dumps the HTML for easy comparison)
```

Reply:
I like this change but it looks like we could keep the length check using:
```js
expect(wrapper).to.have.exactly(1).descendants('video');
```

Reply:
I like it